java/Cherepnin_Oleh/src/com/zoolatech/lecture10/tasks/Task.java

```diff
@@ -0,0 +1,29 @@
+package com.zoolatech.lecture10.tasks;
+
+import java.util.Scanner;
+
+/**
+ * Given a directory called db with 3 files:
+ * names (Ex. of content Alex, John, Maria)
+ * surnames (Scott, Fung, Chan)
+ * positions (Delivery Manager, Senior Java Dev, Product Manager)
+ * <p>
+ * When user starts app it can see in terminal list of files without content. User can type file name like
+ * "names" and see content of the file on terminal.
+ * If user type invalid filename app should write message to err stream, draw file names again and write invalid
+ * filename to a new file called "error.log" (ideally data should be appended).
+ */
+
+public class Task {
+    public static void main(String[] args) {
+        FileReader fileReader = new FileReader("src/com/zoolatech/lecture10/tasks/db/");
+        Scanner scanner = new Scanner(System.in);
+
+        do {
+            System.out.println("Input file name (names, surnames, positions):");
+            String fileName = scanner.next();
+            fileReader.readFile(fileName);
+            System.out.println("Repeat operation (press 'Y' to repeat):");
+        } while (scanner.next().equalsIgnoreCase("Y"));
```

Comment:
nit: not about this case but try to use this format "Y".equalsIgnoreCase(scanner.next()). It looks the same but the constant cannot be null. If you use this approach all the time once it will safe you time on fixing null pointer exception